integration_test/testgenerated/client/api/servers.conjure.go

```diff
@@ -146,3 +150,12 @@ func (t *testServiceHandler) HandleBinary(rw http.ResponseWriter, req *http.Requ
 	rw.Header().Add("Content-Type", codecs.Binary.ContentType())
 	return codecs.Binary.Encode(rw, respArg)
 }
+
+func (t *testServiceHandler) HandleMaybeBinary(rw http.ResponseWriter, req *http.Request) error {
+	respArg, err := t.impl.MaybeBinary(req.Context())
+	if err != nil {
+		return err
+	}
+	rw.Header().Add("Content-Type", codecs.Binary.ContentType())
+	return codecs.Binary.Encode(rw, respArg)
```

Comment:
I think this might break here because `Binary.Encode` is expecting an object of type `Reader` not of type `*Reader`. Working on the server code fix.